scripts/build_utils.sh

```diff
@@ -2,7 +2,7 @@
 
 set -ex
 
-VENV="$WORKSPACE/venv/bin"
+VENV="/tmp/jenkins-venv/bin"
```

Comment:
Can we use `TEMPVENV=$(mktemp -td venv.XXXXXXXXXX)` instead of hard-coding `/tmp/jenkins-venv` ? hard-coded `/tmp` paths make Shadowman weep